cmd/builder/internal/command.go

```diff
@@ -85,6 +86,7 @@ configuration is provided, ocb will generate a default Collector.
 	cmd.Flags().StringVar(&cfgFile, "config", "", "build configuration file")
 
 	// the distribution parameters, which we accept as CLI flags as well
+	cmd.Flags().BoolVar(&cfg.SkipGenerate, skipGenerateFlag, false, "Whether builder should skip generating go code (default false)")
```

Comment:
If it skips generating go code, what does it do?

Reply:
Depending on the other options (SkipCompilation and SkipGetModules), it can fetch modules and or build.

Reply:
But that is a "go" command. If anything this tool should not build at all.

Reply:
You suggest to remove building from the builder ?
Besides, do you know why it's possible to skip module sync but still compiling ? I guess that's also an invalid use case since the go.mod file is not in sync with the generated code in that case.